components/generic/RelatedCollections.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <b-container
+    v-if="relatedCollections.length > 0"
+    data-qa="related collections"
+  >
+    <h2 class="related-heading text-uppercase mt-4 mb-2">
+      {{ $t('relatedCollections') }}
+    </h2>
+    <RelatedChip
+      v-for="relatedCollection in relatedCollections"
+      :id="relatedCollection.id"
+      :key="relatedCollection.id"
+      :link-gen="suggestionLinkGen"
+      :title="relatedCollection.prefLabel[$i18n.locale]"
+    />
+  </b-container>
+</template>
+
+<script>
+  import RelatedChip from './RelatedChip';
+  import { getEntitySuggestions, getEntityTypeHumanReadable, getEntitySlug } from '../../plugins/europeana/entity';
+  import { mapGetters } from 'vuex';
+
+  export default {
+    name: 'RelatedCollections',
+
+    components: {
+      RelatedChip
+    },
+
+    props: {
+      query: {
+        type: String,
+        default: ''
+      }
+    },
+
+    data() {
+      return {
+        relatedCollections: []
+      };
+    },
+
+    computed: {
+      ...mapGetters({
+        apiConfig: 'apis/config'
+      })
+    },
+
+    watch: {
+      query() {
+        this.$fetch();
+      }
+    },
+
+    fetch() {
+      this.getSearchSuggestions(this.query);
+    },
+
+    mounted() {
+      this.$fetch();
+    },
+
+    methods: {
+      async getSearchSuggestions(query) {
+        const suggestions = await getEntitySuggestions(query, {
```

Comment:
Suggestions are showing even if the query is empty, but should not.

Resolve with the logic in the diff in the following comment.